compiler/luci/pass/src/CircleOptimizerUtils.cpp

```diff
@@ -97,4 +98,23 @@ bool has_dynamic_shape(const loco::Node *node)
   return false;
 }
 
+std::vector<loco::Node *> alive_nodes(loco::Graph *g)
```

Comment:
Is `alive` represent this method? Please explain.

Reply:
I thought that `alive` represent this method because it enumerate all the nodes required to export graph.
If it seems strange, maybe `luci::active_nodes` can be applied?